lua/octo/pickers/telescope/entry_maker.lua

```diff
@@ -4,6 +4,54 @@ local utils = require "octo.utils"
 
 local M = {}
 
+-- Symbols found with "Telescope symbols"
+local icons = {
```

Comment:
I am not 100% familiar with lua doctyping, and struggle to find information on this, but is it possible to type items in this to be a table with first element string, and second element a string enum representing the colors ?

If so, we could type this array, as well as the return of `get_icon` function

Reply:
I struggle to find that resourse as well. I will give it a shot